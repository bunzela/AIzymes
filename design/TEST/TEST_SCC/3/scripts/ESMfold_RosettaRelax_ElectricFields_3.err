Lmod has detected the following error: These module(s) or extension(s) exist
but cannot be loaded as requested: "fftw/3.3.10-cuda-12.1.1",
"openmpi/4.1.6-cuda-12.1.1", "netcdf-c/4.9.2-cuda", "cuda"
   Try: "module spider fftw/3.3.10-cuda-12.1.1 openmpi/4.1.6-cuda-12.1.1
netcdf-c/4.9.2-cuda cuda" to see how to load the module(s).



Some weights of EsmForProteinFolding were not initialized from the model checkpoint at facebook/esmfold_v1 and are newly initialized: ['esm.contact_head.regression.bias', 'esm.contact_head.regression.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
sed: can't read /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ESMFold/7vuu_3_aligned.pdb: No such file or directory
sed: can't read /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ESMFold/7vuu_3_lig.pdb: No such file or directory
cat: /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ESMFold/7vuu_3_aligned.pdb: No such file or directory
cat: /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ESMFold/7vuu_3_lig.pdb: No such file or directory

ERROR: Cannot determine file type. Current supported types are: PDB, CIF, SRLZ, MMTF
ERROR:: Exit from: src/core/import_pose/import_pose.cc line: 379

[ ERROR ]: Caught exception:


File: src/core/import_pose/import_pose.cc:379
[ ERROR ] UtilityExitException
ERROR: Cannot determine file type. Current supported types are: PDB, CIF, SRLZ, MMTF




AN INTERNAL ERROR HAS OCCURED. PLEASE SEE THE CONTENTS OF ROSETTA_CRASH.log FOR DETAILS.



---------------------------------------------------------------
[ ERROR ]: Error(s) were encountered when running jobs.
1 jobs failed;
Check the output further up for additional error messages.
---------------------------------------------------------------

mv: cannot stat '/mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/7vuu_3_input_0001.pdb': No such file or directory
sed: can't read 7vuu_RosettaRelax_3.pdb: No such file or directory
sed: can't read /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/7vuu_RosettaRelax_3.pdb: No such file or directory
/scratch-scc/projects/scc_mmtm_bunzel/amber24/bin/teLeap: error while loading shared libraries: libnetcdf.so.19: cannot open shared object file: No such file or directory
mv: cannot stat 'leap.log': No such file or directory
Traceback (most recent call last):
  File "/mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/../../../src/FieldTools.py", line 659, in <module>
    main()
  File "/mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/../../../src/FieldTools.py", line 576, in main
    Trajectory, Charges, Names = fkt_Load_Trajectory(arg_nc,arg_param,arg_TIP4P)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/../../../src/FieldTools.py", line 251, in fkt_Load_Trajectory
    parm = pmd.load_file(arg_parm)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch-scc/projects/scc_mmtm_bunzel/amber24/lib/python3.12/site-packages/parmed/formats/registry.py", line 166, in load_file
    raise FileNotFoundError('%s does not exist' % filename)
FileNotFoundError: /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ElectricFields/7vuu_RosettaRelax_3.parm7 does not exist
