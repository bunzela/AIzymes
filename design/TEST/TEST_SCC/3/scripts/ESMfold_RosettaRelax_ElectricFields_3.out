/mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3
********  (C) Copyright Rosetta Commons Member Institutions.  ***************
* Use of Rosetta for commercial purposes may require purchase of a license. *
********  See LICENSE.md or email license@uw.edu for more details. **********
core.init: Checking for fconfig files in pwd and ./rosetta/flags 
core.init: Rosetta version: rosetta.source.release-371 r371 2024.09+release.06b3cf8 06b3cf8ad0940d628690d0ed6fa2009d72ad2b44 https://www.rosettacommons.org 2024-03-07T14:59:22.624665
core.init: command: /scratch-scc/projects/scc_mmtm_bunzel/rosetta.source.release-371/main/source//bin/rosetta_scripts.linuxgccrelease -s /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/RosettaRelax/7vuu_3_input.pdb -extra_res_fa /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/Input/5TS.params -parser:protocol /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/scripts/RosettaRelax_3.xml -out:file:scorefile /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/score_RosettaRelax.sc -nstruct 1 -ignore_zero_occupancy false -corrections::beta_nov16 true -run:preserve_header true -overwrite
basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-1357232515 seed_offset=0 real_seed=-1357232515
basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-1357232515 RG_type=mt19937
core.init: Resolved executable path: /scratch-scc/projects/scc_mmtm_bunzel/rosetta.source.release-371/main/source/build/src/release/linux/4.18/64/x86/gcc/8/default/rosetta_scripts.default.linuxgccrelease
core.init: Looking for database based on location of executable: /scratch-scc/projects/scc_mmtm_bunzel/rosetta.source.release-371/main/database/
protocols.jd2.PDBJobInputter: Instantiate PDBJobInputter
protocols.jd2.PDBJobInputter: PDBJobInputter::fill_jobs
protocols.jd2.PDBJobInputter: pushed /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/RosettaRelax/7vuu_3_input.pdb nstruct index 1
protocols.evaluation.ChiWellRmsdEvaluatorCreator: Evaluation Creator active ... 
protocols.jd2.PDBJobInputter: PDBJobInputter::pose_from_job
protocols.jd2.PDBJobInputter: filling pose from PDB /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/RosettaRelax/7vuu_3_input.pdb
core.chemical.GlobalResidueTypeSet: Finished initializing fa_standard residue type set.  Created 986 residue types
core.chemical.GlobalResidueTypeSet: Total time to initialize 1.62269 seconds.
core.import_pose.import_pose: File '/mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/RosettaRelax/7vuu_3_input.pdb' automatically determined to be of type UNKNOWN
Error: [ ERROR ] ERROR: Exception caught by JobDistributor while trying to get pose from job '7vuu_3_input_0001'
Error: [ ERROR ] Treating failure as bad input; canceling similar jobs
protocols.jd2.FileSystemJobDistributor: job failed, reporting bad input; other jobs of same input will be canceled: 7vuu_3_input_0001
protocols.jd2.JobDistributor: no more batches to process... 
protocols.jd2.JobDistributor: 1 jobs considered, 1 jobs attempted in 53 seconds
-nc              Trajectory file      :  /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ElectricFields/7vuu_RosettaRelax_3.rst7
-parm            Parameter file       :  /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ElectricFields/7vuu_RosettaRelax_3.parm7
-out             Output file          :  /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/3/ElectricFields/7vuu_RosettaRelax_3_fields.pkl
-target          Field target         :  /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/parent/field_target.dat
-arg_solvent     Non-protein residues :  WAT
-exclude_atoms   Atoms excluded       : Full residue defined in the first Atom for each Target.

Field calculation RUNNING :  2025-02-18 19:08:57.543557
